packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -143,50 +119,27 @@ export async function enqueueLinks(options: EnqueueLinksOptions): Promise<QueueO
         transformRequestFunction: ow.optional.function,
     }));
 
-    if (baseUrl && page) log.warning('The parameter options.baseUrl can only be used when parsing a Cheerio object. It will be ignored.');
-
     // Construct pseudoUrls from input where necessary.
-    const pseudoUrlInstances = constructPseudoUrlInstances(pseudoUrls as PseudoUrlInput[] || []);
+    let parsedPseudoUrls: PseudoUrlInput[];
+
+    if (Array.isArray(pseudoUrls)) {
+        parsedPseudoUrls = pseudoUrls;
+    } else if (pseudoUrls) {
+        parsedPseudoUrls = [pseudoUrls];
+    } else {
+        parsedPseudoUrls = [];
+    }
```

Comment:
why is this needed? the ow validation seems to allow only null and an array, so the middle branch seems not reachable